res/openvpn-gui-res-ru.rc

```diff
@@ -134,17 +134,18 @@ BEGIN
     GROUPBOX "Интерфейс пользователя", 201, 6, 12, 235, 30
     LTEXT "Язык:", ID_TXT_LANGUAGE, 17, 25, 23, 12
     COMBOBOX ID_CMB_LANGUAGE, 42, 23, 186, 400, CBS_DROPDOWNLIST | WS_TABSTOP
-    GROUPBOX "Startup", 202, 6, 47, 235, 30
-    AUTOCHECKBOX "Launch on Windows startup", ID_CHK_STARTUP, 17, 59, 200, 12
 
-    GROUPBOX "Preferences", 202, 6, 82, 235, 90
-    AUTOCHECKBOX "Append to log", ID_CHK_LOG_APPEND, 17, 95, 60, 10
-    AUTOCHECKBOX "Show script window", ID_CHK_SHOW_SCRIPT_WIN, 17, 110, 200, 10
-    AUTOCHECKBOX "Silent connection", ID_CHK_SILENT, 17, 125, 200, 10
-    LTEXT "Show Balloon", ID_TXT_BALLOON, 17, 140, 100, 10
-    AUTORADIOBUTTON "On connect", ID_RB_BALLOON1, 28, 155, 50, 10, WS_GROUP | WS_TABSTOP
-    AUTORADIOBUTTON "On connect/reconnect", ID_RB_BALLOON2, 86, 155, 90, 10
-    AUTORADIOBUTTON "Never", ID_RB_BALLOON0, 181, 155, 40, 10
+    GROUPBOX "Запуск", 202, 6, 47, 235, 30
+    AUTOCHECKBOX "Запускаться при старте Windows", ID_CHK_STARTUP, 17, 59, 200, 12
+
+    GROUPBOX "Настройки", 202, 6, 82, 235, 90
+    AUTOCHECKBOX "Дописывать, а не перезаписывать, лог", ID_CHK_LOG_APPEND, 17, 95, 60, 10
+    AUTOCHECKBOX "Отображать окно скрипта", ID_CHK_SHOW_SCRIPT_WIN, 17, 110, 200, 10
+    AUTOCHECKBOX "«Тихое» подключение", ID_CHK_SILENT, 17, 125, 200, 10
+    LTEXT "Показывать балун", ID_TXT_BALLOON, 17, 140, 100, 10
```

Comment:
@selvanair yes, "quiet" is used here. However the question was about "balloon". In my opinion, a minority of users heard this word but more people are aware of generic "popup". That's a question for @chipitsine and lev.